js/ui/grid_core/ui.grid_core.header_filter.js

```diff
@@ -511,6 +510,7 @@ export const headerFilterModule = {
                 height: 325,
                 allowSearch: false,
                 searchTimeout: 500,
+                showRelevantValues: true,
```

Comment:
We should think about option names. The showRelevantValues option name does not say that it is applied to lookup columns only. In the PivotGrid this option is applied to all fields.